src/controller.js

```diff
@@ -0,0 +1,18 @@
+class Controller{
+    constructor(){
+        this.initializeSea();
+    }
+    initializeSea() {
```

Comment:
Indentation is all over the place in this file.

Use the format document option within VS Code.

<img width="464" alt="Screen Shot 2020-11-21 at 13 33 24" src="https://user-images.githubusercontent.com/1476886/99878567-3d982200-2bfe-11eb-8fb6-5dcd707e89f1.png">